### ISSUE TITLE
Error with multy-company rules

### ISSUE THREAD

**Issue by PabloGonzalezRivero** [OCA/multi-company#586]
module: partner_multi_company
version: 15.0

Due to a confusion, I modified one of the permissions of the multicompany module, and now all users can see all contacts, even those from other companies. I've tried uninstalling the multicompany modules entirely and reinstalling them to restore the permissions, but it hasn't worked. Nothing I've tried has worked. What could I do?

**Comment by pedrobaeza** [comment]
Without knowing what you "modify", nothing can be done.

**Comment by PabloGonzalezRivero** [comment]
You won't believe it, while searching for the error (which I couldn't remember because it was a long time ago and I couldn't find it), I discovered what record rule had been modified, and I've restored it to its original state. The issue was caused by altering the 'res.partner company' rule. I've set it to be global, and now it works correctly. I've been dealing with this problem for over 5 months and invested a lot of time in this error, only to finally find the solution this way. I'm sorry for having wasted your time.
